CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+To develop the Java Client and ultimately submit a pull request, you'll first need to be able to build the client and 
+run the tests locally. 
+
+Please see [this guide](.github/CONTRIBUTING.md) for information on creating and submitting a pull request.
+
+To build the client locally, complete the following steps:
+
+1. Clone this repository on your machine.
+2. Choose the appropriate branch (usually develop)
+3. Ensure you are using Java 8 or Java 11
+4. Verify that you can build the client by running `./gradlew build -x test`
+
+"Running in the tests" in the context of developing and submitting a pull request refers to running the tests found
```

Comment:
Nitpicky, but are these the right words? Should it simply be "Running the tests"?

Reply:
Nitpicky in a guide is good.